var/spack/repos/builtin/packages/elemental/package.py

```diff
@@ -145,7 +148,7 @@ def cmake_args(self):
             libfortran = LibraryList(mpif77('--print-file-name',
                                             'libgfortran.%s' % dso_suffix,
                                             output=str))
-        if libfortran:
+        if 'libfortran' in locals():
```

Comment:
@bvanessen I have a similar preference to what I mention in https://github.com/spack/spack/pull/6335#discussion_r164015725. That being said I'm not against `locals()` since it is correct but just slightly harder to understand than falling back to `libfortran=None`. Let me know your opinion.